### ISSUE TITLE
[SPIKE] Analyze storage requirements for prometheus

### ISSUE THREAD

**Issue by pkavajin** [kubermatic/kubermatic#1725]
We should analyze storage usage of prometheus to gain insight about how much storage is used on average per user cluster, so we can make meaningful recommendations when setting up environments.

Result should be a documented statement about the storage requirements.
Something like: "per cluster we'll need 5 GB of storage for metrics if the retention is 2 weeks"

**Comment by xrstf** [comment]
* Retention period is just *12 hours*.
* Prometheus writes WAL segment files of 256MiB and I got mine to create two before garbage collecting the first one. The pure data was roughly 50MiB before the pod was rescheduled and lost all of its data. This means that we should roughly plan for 2*256MiB + 50MiB = 562MiB. Let's say 600-700 MiB.

TL;DR: 12 hours, ~700MiB disk, ~150MiB RAM.

**Comment by xrstf** [comment]
I had a cluster running on cloud and a dedicated Prometheus in the seed. After nearly three weeks, the numbers say this:

* The seed level's Prometheus disk usage increases slowly over the time of two weeks to ~81MiB and then stays there.
* The user cluster Prometheus varies greatly between 30 and 500 MiB, depending on the WAL size. I don't know if a smaller disk would make Prometheus process the WAL quicker.

TL;DR:

* Seed Prometheus: 100 MiB per user cluster
* User Prometheus: on average 250 MiB

**Comment by xrstf** [comment]
Fixed by kubermatic/docs#104.
